### ISSUE TITLE
Hitches with large clothing asset IFRs

### ISSUE THREAD

**Issue by Keechen12** [citizenfx/fivem#1951]
Players on my server are having major issues with hitches when entering areas with groups of players, I've done some testing and found that when IFRs for clothing assets, in particular, are being requested in bulk, the game freezes. The more requested assets, the longer the freeze.

I was wondering if this is something that can be fixed, potentially by staggering large asset requests so they don't cause long freezes, or via some other complex process, such as requesting different types of clothing components at different proximities.

OR 

If it's an issue purely with the assets themselves, which would be resolved by removing them or decreasing their file size.

Clip: https://cdn.discordapp.com/attachments/1000172223757299824/1098377025577812078/Untitled_video_1.mp4

**Comment by blattersturm** [comment]
Instead of a 'clip' with no reproduction steps, can you provide a .etl trace at least?

Also, since a common cause for this also is VRAM exhaustion (which can't be fixed with client changes), how much VRAM does the user have here and what's the 'extended budget scale' set to?

**Comment by Keechen12** [comment]
Yeah sorry I'll get you a trace tomorrow as soon as I figure out how to generate one. I wasn't entirely sure what might be useful for you guys. Is there any other information I can provide that would be helpful?

**Comment by gottfriedleibniz** [comment]
For generating an ETW trace, see https://support.cfx.re/hc/en-us/articles/8366604193436-Creating-an-ETW-trace.

If you are not having issues anymore - you can close this issue.
